test/logger-log.js

```diff
@@ -0,0 +1,449 @@
+'use strict'
```

Comment:
This needs a test that flushes on a timer so we can be sure that multiple lines are sent in 1 call

Reply:
Added.  This reveals something else to go onto LOG-6626 having to do with calling `.log` multiple times.  Callbacks on the successive calls are swallowed, but we should return `Buffered` or something as a response to those.